tests/Http/InputTest.php

```diff
@@ -179,6 +181,20 @@ public function testIsJsonExpected(): void
         $this->container->bind(ServerRequestInterface::class, $request);
 
         $this->assertTrue($this->input->isJsonExpected());
+
+        $request = new ServerRequest(
+            [],
+            [],
+            'http://domain.com/hello-world',
+            'GET',
+            'php://input',
+            [
+                'Accept' => 'application/vnd.api+json'
+            ]
+        );
+        $this->container->bind(ServerRequestInterface::class, $request);
+
+        $this->assertTrue($this->input->isJsonExpected());
```

Comment:
Can you please add also 1 negative test case (assertFalse without adding a specific json type)?